README.md

```diff
@@ -129,8 +130,24 @@ You can deploy your Palico app to any cloud provider using Docker or use our man
 
 Learn more from [docs](https://docs.palico.ai/guides/client_sdk).
 
+## ⭐ Star History
```

Comment:
Let's remove this for now. I'll add it later.

Reply:
OK